src/components/charts/shared/Tooltip.jsx

```diff
@@ -85,9 +87,13 @@ class Tooltip extends React.PureComponent {
   renderValue(accessor, data, unit) {
     return (
       <>
-        <span>{getValue(accessor, data)}</span>
+        <span>
+          <TranslatedValue {...{ accessor, data }} />
```

Comment:
switched to using the `TranslatedValue` component instead of the `getValue` utility method.